module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -282,4 +283,57 @@ public function holdAction()
         $view->setTemplate('record/hold');
         return $view;
     }
+
+    /**
+     * Email the successful request to staff as a notification.
+     *
+     * @param array $holdDetails Details about the successful hold request
+     *
+     * @return void
+     */
+    protected function emailRequestPlaced($holdDetails)
```

Comment:
VuFind calls these user actions both holds (too specific) and requests (maybe too general).  I use requests where possible but holds where the term is already used i.e. in that config section.